statsmodels/duration/hazard_regression.py

```diff
@@ -1172,19 +1168,14 @@ def baseline_cumulative_hazard_function(self, params):
         return cumhaz_f
 
     def predict(self, params, exog=None, cov_params=None, endog=None,
-                strata=None, offset=None, pred_type="lhr"):
-        # docstring attached below
+                strata=None, offset=None, pred_type="lhr", 
+                return_object=False, **kwargs):
 
         pred_type = pred_type.lower()
-        if pred_type not in ["lhr", "hr", "surv", "cumhaz"]:
+        if pred_type not in ["lhr", "hr", "surv", "cumhaz", "qsurv"]:
```

Comment:
Yeah, I do think they got crossed, I'm not sure what the best way is to fix that.